test/deploy-eks-cluster.sh

```diff
@@ -16,8 +16,8 @@ if [[ -z "${ECR_REPOSITORY}" ]]; then
 fi
 
 if [[ -z "${EKS_CLUSTER_K8_VERSION}" ]]; then
-  echo "EKS_CLUSTER_K8_VERSION not set. Changing to 1.22"
-  export EKS_CLUSTER_K8_VERSION="1.22"
+  echo "EKS_CLUSTER_K8_VERSION not set. Changing to 1.26"
```

Comment:
does it have to be 1.28 like we have in .env file?

Reply:
1.28 is kubectl version not k8s